frontend/src/views/ExtractView.vue

```diff
@@ -82,6 +82,13 @@ const extractFrame = async () => {
     selectedFrame.value = framesList.value.indexOf(data.value[0]);
   }
 };
+
+// Change player to frame of clicked of element
+const clickFrame = frameName => {
+  const regex = /\d+/;
+  const frameNumber = regex.exec(frameName);
```

Comment:
The regex here is so simple you can just inline it, if it was more complex or used multiple times then I'd assign it to a variable and give it some intrinsic documentation e.g. `const regexDigit ...`
```suggestion
  const frameNumber = /\d+/.exec(frameName);
```